### PR TITLE
Antd update

### PR DESCRIPTION
**Pull request by david-pirkle**
This updates Ant Design to the latest version. Also cleaned up the code some.

### CODE REVIEW

File: frontend/src/containers/SearchTable.jsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ export default class SearchTable extends Component {
             isLoading: true
         }, async () => {
             const url = this.state.geneSearch ?
-                `http://localhost:8000/variants/?page=${this.state.currentPage}&geneSearch=${this.state.geneSearch}` :
+                `http://localhost:8000/variants/?page=${this.state.currentPage}&search=${this.state.geneSearch}` :
```
Comment (andreas-invitae) [inline frontend/src/containers/SearchTable.jsx:88]: not sure if this one was intentional?

Comment (david-pirkle) [reply frontend/src/containers/SearchTable.jsx:88]: Yes, "search" is the parameter that the SearchFilter uses. One of the changes I made was to replace the search implementation in the endpoint with SearchFilter from the DRF.

**Review by teganduong — APPROVED** [review]

File: backend/variant_search/views.py [hunk 2]
```diff
@@ -11,29 +12,23 @@ class VariantViewSet(viewsets.ModelViewSet):
     # Apply and then filter by the gene param if set
     queryset = Variant.objects.exclude(gene='')
     serializer_class = VariantSerializer
+    filter_backends = (filters.SearchFilter,)
```
Comment (RohitTalwalkar) [inline backend/variant_search/views.py:15]: hmm, I think we'd purposely left the query / filtering stuff out, but were candidates finding issues with the current setup?

File: frontend/src/containers/SearchTable.jsx [hunk 3]
```diff
@@ -105,27 +105,27 @@ export default class SearchTable extends Component {
         this.setState({
             currentPage: newPage
         }, () => this.loadDataForPage(this.state.currentPage));
-    }
+    };
```
Comment (RohitTalwalkar) [inline frontend/src/containers/SearchTable.jsx:108]: oh, I wonder if we should add a linter to the project...

**Review by RohitTalwalkar — APPROVED** [review]
LGMT

File: backend/variant_search/views.py (hunk 2 above)
Comment (david-pirkle) [reply backend/variant_search/views.py:15]: When I get into pair programming with candidates, one of the things I look for is whether they are using built-in capabilities of the stack we've given them to work with. Even if someone isn't terribly familiar with DRF, by leveraging those capabilities in the example app, I think it's a clue to them that these exercises can use other built-in capabilities. For example, the entirety of the backend changes needed for the sorting exercise can be implemented by changing this one line to:
`filter_backends = (filters.SearchFilter, filters.OrderingFilter)`

Comment (david-pirkle) [reply backend/variant_search/views.py:15]: As far as that comment that "we'd purposely left the query / filtering stuff out", the existing example does implement searching, so nothing has changed as far as the basic endpoint support for searching by gene. This change only moves the implementation of that searching out of the endpoint, by leveraging `filters.SearchFilter`.

File: frontend/src/containers/SearchTable.jsx (hunk 3 above)
Comment (david-pirkle) [reply frontend/src/containers/SearchTable.jsx:108]: Good idea. Another PR for that?

**Review by andreas-invitae — APPROVED** [review]
LGTM

**Pull request merged**